### DISCUSSION TITLE
Wallabag url?

### DISCUSSION THREAD

**Discussion by pyromaker** [wallabag/wallabag#6202] [Q&A]
Hello, I am trying to connect wallabag with another service and it asks me the server url. What should I put there?

I created an API and put in the the other details client ID client secret user pass.
Sorry I am very new to this one. Only today I found about wallabag

**Comment by j0k3r** [comment]
> What should I put there?

Uh, the wallabag url of your instance?
I mean, are you selfhosted or are you using wallabag.it?
